//
// PROJECT:         Aspia
// FILE:            category/category_windows_device.proto
// LICENSE:         GNU Lesser General Public License 2.1
// PROGRAMMERS:     Dmitry Chapyshev (dmitry@aspia.ru)
//

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

package aspia.proto;

message WindowsDevice
{
    message Item
    {
        string friendly_name  = 1;
        string description    = 2;
        string driver_version = 3;
        string driver_date    = 4;
        string driver_vendor  = 5;
        string device_id      = 6;
    }

    repeated Item item = 1;
}
